Assets/Scripts/AutomatedTest/PersistentDataLogIn.cs

```diff
@@ -1,60 +1,63 @@
-﻿using UnityEngine;
-using System.Collections;
-using UnityEngine.SceneManagement;
+﻿using System.Collections;
 using System.Collections.Generic;
+using UnityEngine;
 using UnityEngine.Networking;
+using UnityEngine.SceneManagement;
 
-public class PersistentDataLogIn : MonoBehaviour
+namespace MSP2050.Scripts
 {
-	private int sessionId;
-	private int countryId;
-	private string userName;
-	private MspGlobalData mspGlobalData;
-	Dictionary<int, Team> teams;
-
-	public void Initialize(int countryId, string userName, MspGlobalData mspGlobalData, Dictionary<int, Team> teams)
+	public class PersistentDataLogIn : MonoBehaviour
 	{
-		//NetworkForm form = new NetworkForm();
+		private int sessionId;
+		private int countryId;
+		private string userName;
+		private MspGlobalData mspGlobalData;
+		Dictionary<int, Team> teams;
 
-		//form.AddField("layer", planlayer.ID);
-		//form.AddField("object", IssueManager.Serialize(planlayer));
-		//form.AddField("active", 1);
+		public void Initialize(int countryId, string userName, MspGlobalData mspGlobalData, Dictionary<int, Team> teams)
+		{
+			//NetworkForm form = new NetworkForm();
```

Comment:
Could this commented code be relevant in the future?
If not I'd remove it for good!